Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -97,11 +95,35 @@ class SentryUserFeedbackForm: UIViewController {
         addedScreenshot(image: image)
         return
 #else
-        let imagePickerController = UIImagePickerController()
-        imagePickerController.delegate = self
-        imagePickerController.sourceType = .photoLibrary
-        imagePickerController.allowsEditing = true
-        present(imagePickerController, animated: config.animations)
+        
+        func presentPicker() {
+            let imagePickerController = UIImagePickerController()
+            imagePickerController.delegate = self
+            imagePickerController.sourceType = .photoLibrary
+            imagePickerController.allowsEditing = true
+            DispatchQueue.main.async {
+                self.present(imagePickerController, animated: self.config.animations)
+            }
+        }
+        
+        let status = PHPhotoLibrary.authorizationStatus()
```

Comment:
`m`: `authorizationStatus()` is [deprecated](https://developer.apple.com/documentation/photos/phphotolibrary/authorizationstatus()), we should start using [authorizationStatus(for:)](https://developer.apple.com/documentation/photos/phphotolibrary/authorizationstatus(for:)) starting with `iOS 14.0`.

Also

Reply:
I can add this back in. I originally had it but it was crashing because I hadn't added the infoplist property. Although before I realized that was the cause, I thought there was something going on with this API 😆

Reply:
I'll also do this in the separate photo picker PR